src/main/java/com/zendesk/maxwell/schema/Table.java

```diff
@@ -8,33 +8,56 @@
 import com.zendesk.maxwell.schema.columndef.ColumnDef;
 import com.zendesk.maxwell.schema.columndef.StringColumnDef;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 public class Table {
-	private final List<ColumnDef> columnList;
-	int pkIndex;
-	private String name;
-	private String database;
-	private final String charset;
-	private List<String> pkColumnNames;
+	public String database;
+	@JsonProperty("table")
+	public String name;
+	private List<ColumnDef> columnList;
+	public String charset;
+	@JsonProperty("primary-key")
+	public List<String> pkColumnNames;
+
 	private HashMap<String, Integer> columnOffsetMap;
+	@JsonIgnore
+	public int pkIndex;
 
+	public Table() { }
 	public Table(String database, String name, String charset, List<ColumnDef> list, List<String> pks) {
 		this.database = database;
 		this.name = name;
 		this.charset = charset;
-		this.columnList = list;
+		this.setColumnList(list);
 
 		if ( pks == null )
 			pks = new ArrayList<String>();
 
 		this.setPKList(pks);
-
-		renumberColumns();
 	}
 
+	@JsonProperty("columns")
 	public List<ColumnDef> getColumnList() {
 		return columnList;
 	}
 
+	@JsonProperty("columns")
```

Comment:
is this correct, JsonProperty on a setter? (and it already defines `getColumnList ` as 'columns' above)